hlink/tests/training_test.py

```diff
@@ -112,6 +112,15 @@ def test_all_steps(
     assert row.state_distance_imp.round(0) == 1909
 
     training.run_step(3)
+    tf = spark.table("training_feature_importances").toPandas()
+    for var in training_conf["training"]["independent_vars"]:
+        assert not tf.loc[tf["feature_name"].str.startswith(f"{var}", na=False)].empty
+    assert all([col in ['feature_name', 'coefficient_or_importance'] for col in tf.columns])
```

Comment:
Here we should be able to use a generator expression instead of a list comprehension, which means we can drop the outermost pair of `[`s. This is not a big deal, but a nice style thing.